app/assets/stylesheets/admin.scss

```
@@ -0,0 +1,65 @@
+@import 'var_shop';
+.admin_container{
+  width: 80%;
+  margin: auto;
+  background-color: #f8f9fa;
+  margin-top: 40px;
+}
+.page-title {
```

Chỗ này cách ra nè huynh ơi
```suggestion
}

.page-title {
```

---

rồi chưa fix nè